.github/workflows/hpu-gaudi2.yml

```diff
@@ -5,10 +5,11 @@ on:
   schedule:
     - cron: "0 0 * * *"
   pull_request:
-    paths:
-      - ".github/workflows/hpu-gaudi2.yml"
-      - "accelerator/hpu_accelerator.py"
-
+    paths-ignore:
```

Comment:
Hi @vshekhawat-hlab - I'm not sure we want to run on every commit, we run nightly for this reason, to reduce the total scope of tests run on each PR, and should be able to identify any regression that way.  Do you feel this needs to be run on all PRs?  My concern is there isn't enough capacity (1 machine) supporting this test, so we will queue on waiting for these tests to pass.

Reply:
Thank you, @loadams , for notifying us of the issues for running Gaudi2 CI with every commit. Let me check and get back to you on this. We aim to increase coverage for Gaudi2 CI to reduce the pain of bisection.

Reply:
Certainly, I understand that bisecting commits can be annoying, though we don't tend to merge that many commits per day.  But we can certainly add more path filters too